tasks/qlogic_network_adapter.yml

```diff
@@ -0,0 +1,14 @@
+---
+- assert: { that: "ansible_system_vendor == 'HP'" }
```

Comment:
I don't know the QLogic seems so weird, Dell seems also to be selling the same card. Maybe there is a better assert?

lspci |grep 'NetXtreme II BCM57810'
05:00.0 Ethernet controller: Broadcom Limited NetXtreme II BCM57810 10 Gigabit Ethernet (rev 10)

Reply:
Added some asserts